### PR TITLE
Improved function syntax 

### PR DESCRIPTION
**Pull request by howardbaik**
Added titles to every function and inline comments to explain what each function is doing.

### CODE REVIEW

File: R/aaa_utils.R [hunk 1]
```diff
@@ -332,7 +359,7 @@ get_slide_id = function(x) {
 
 #' @export
 #' @rdname get_slide_id
-make_slide_url = function(x) {
+make_slide_url <- function(x) {
```
Comment (cansavvy) [inline R/aaa_utils.R:362]: You may want to consider (probably in a different PR so as to keep this focused on what you've done already) running this through styler so it styles the code for you. https://styler.r-lib.org/articles/introducing_styler.html

**Review by cansavvy — APPROVED** [review]
These are great! Thanks for doing all this tedious work! I think you are setting up the next chunk of the project with a great foundation by taking the time to do this annotation!

**Comment by cansavvy** [comment]
Ah, the checks are failing here because this also needs the version bumps akin to what we did here: https://github.com/jhudsl/ari/pull/43

Do you want to sneak that in here?

File: R/aaa_utils.R (hunk 1 above)
Comment (howardbaik) [reply R/aaa_utils.R:362]: That's a great idea. I'll run this through styler in a separate PR.

**Comment by howardbaik** [comment]
I just bumped up the version and deleted the two .yml files for travis and appveyor!

**Comment by cansavvy** [comment]
May also need to bump up  r-lib/actions/check-r-package@v1 ?

**Comment by howardbaik** [comment]
@cansavvy  I bumped up the version from `r-lib/actions/check-r-package@v1` to `r-lib/actions/check-r-package@v2`, but the `R-CMD-check / windows-latest` is still failing, and I have no idea what this error message is saying:

https://github.com/jhudsl/ariExtra/actions/runs/4813676124/jobs/8570505579?pr=5

**Comment by cansavvy** [comment]
One of the tests for one of the functions is failing. We can try to look into why. I'll make a separate issue for that. And we can merge this pr.

**Comment by cansavvy** [comment]
Filed: https://github.com/jhudsl/ariExtra/issues/6

**Pull request merged**